docs/user/SUMMARY.md

```diff
@@ -23,14 +23,18 @@
     - [Testing custom pings](user/pings/testing-custom-pings.md)
 - [Using the experiments API](user/experiments-api.md)
 - [Debugging products using Glean](user/debugging/index.md)
-    - [Glean Debug Ping View](user/debugging/debug-ping-view.md)
     - [Android](user/debugging/android.md)
```

Comment:
I debated changing these to "Kotlin" and "Swift", for standardization sake. The guides are very Android and iOS specific, but then again so is everything else Kotlin / Swift related. Anyways, just a though, but it is weird to call the Kotlin bindings "Kotlin" / "Android" interchangeably and same for "Swift" / "iOS".

Reply:
I agree, it's indeed weird . I think that predates our idea / term of "language bindings"

Reply:
I guess the naming proposal will help reach a consensus here, so I will ignore the weirdness for now.